src/pages/productPage/productPage.tsx

```diff
@@ -1,12 +1,26 @@
 import { useParams } from 'react-router-dom';
 
+import img from '../../assets/images/Rectangle 51.png';
+import { Footer } from '../../components/footer';
+import { Header } from '../../components/header';
+import { ProductCard } from '../../components/productCard';
 import classes from './productPage.module.scss';
 
 export function ProductPage() {
   const { id } = useParams();
   return (
     <div id={id} className={classes.cont}>
-      product
+      <Header promoTitle="FREE SHIPPING ON ALL FULL SUN PLANTS! FEB. 25–28." />
+      <ProductCard
```

Comment:
IMO the ProductCard must be a container (aka Smart component). You can pass product id to this container and fetch data inside of it